src/features/search/pages/modals/VenueModal/useVenueModal.ts

```diff
@@ -72,6 +72,12 @@ const useVenueModal = ({ dismissModal, doAfterSearch }: VenueModalHookProps): Ve
     dismissModal()
   }, [dismissModal, dispatch, doAfterSearch, searchState, selectedVenue, userPosition, venueQuery])
 
+  useEffect(() => {
+    if (searchState.locationFilter.locationType === LocationType.VENUE) {
+      setVenueQuery(searchState.locationFilter.venue.label)
+    }
+  }, [searchState.locationFilter])
```

Comment:
[suggestion] que ce passe-t-il si tu précise `searchState.locationFilter.locationType` ?

Reply:
en dépendances ?

Reply:
Oui, avec aussi searchState.locationFilter.venue.label

Reply:
Pas possible car : Property 'venue' does not exist on type 'LocationFilter'.
  Property 'venue' does not exist on type '{ locationType: LocationType.EVERYWHERE; }'.